frontend/src/components/common/cards/ForestClientCard.vue

```diff
@@ -0,0 +1,73 @@
+<script setup lang="ts">
+import type { FamForestClientStatus } from 'fam-api/dist/model/fam-forest-client-status';
+import { ref, type PropType } from 'vue';
+import Card from 'primevue/card';
+
+const selected = ref('');
```

Comment:
This variable is not in use anymore, let's remove it

Reply:
Thanks, I'll remove it.